docs/source/getting-started/index.rst

```diff
@@ -77,9 +54,61 @@ While the specifics of each tool differ, most tools will pick up your account in
 
    * - Environment variable
      - Value
+   * - ``MYTHX_API_KEY``
```

Comment:
Can we just call this the API key? I'm not sure that everyone knows what JWT is (or that it's relevant).